src/components/Spinner.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import SpinnerImage from '../assets/spinner.svg'
+import styled, { keyframes } from "styled-components";
+
+function Spinner (props) {
+    return <SpinnerContainer>
+        <Rotate>
+            <img src={SpinnerImage} alt={'Loading...'}/>
+        </Rotate>
+    </SpinnerContainer>
+}
+
+const rotate = keyframes`
+  from {
+    transform: rotate(360deg);
+  }
+
+  to {
+    transform: rotate(0deg);
+  }
+`;
+
+const Rotate = styled.div`
+  animation: ${rotate} 1s linear infinite;
+`;
+
+
+const SpinnerContainer = styled.ul`
+   text-align: center;
+   margin-top: 10vh;
+   img {
+    height: 40px;
+   }
+`
+
+export default Spinner;
```

Comment:
new line

Reply:
Yes, it's a new line.

Reply:
We shall consider using husky & lint-staged & prettier to keep our Code Review meaningful. 
https://github.com/kgajowy/community-suggestions-web/blob/master/.lintstagedrc
https://github.com/kgajowy/community-suggestions-web/blob/master/.huskyrc
This is a great topic for different PR which would save us a few moments when talking/writing about formatting issues ;)